adaptdl/adaptdl/torch/_metrics.py

```diff
@@ -45,6 +41,21 @@ def profile_sync_time(sync_time):
 _PREV_REPORT = None
 
 
+def profile_accumulation_commit():
```

Comment:
Make it a flag `profile_step_commit(accumulate=True)` to reduce logic duplication.

Reply:
Fixed